### PR TITLE
minor: fix rustfmt checks

### PR DESCRIPTION
**Pull request by saghm**

### CODE REVIEW

File: src/cmap/background.rs [hunk 1]
```diff
@@ -40,10 +37,12 @@ impl ConnectionPool {
 
         let mut i = 0;
         while i < connection_manager.checked_in_connections.len() {
-            if connection_manager.checked_in_connections[i].is_stale(connection_manager.generation) {
+            if connection_manager.checked_in_connections[i].is_stale(connection_manager.generation)
+            {
```
Comment (patrickfreed) [inline src/cmap/background.rs]: this hanging brace is kind of weird. could we configure rustfmt to break on one of the dot-operators instead?

File: .evergreen/check-rustfmt.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
+#!/bin/bash
 
 set -o errexit
 
 . ~/.cargo/env
-cargo +nightly fmt -- --check
+rustfmt +nightly --unstable-features --check src/**/*.rs
```
Comment (patrickfreed) [inline .evergreen/check-rustfmt.sh]: do we need to keep using the nightly version if we're using `unstable-features`?

Comment (saghm) [reply .evergreen/check-rustfmt.sh]: `--unstable-features` is a nightly-only flag

File: src/cmap/background.rs (hunk 1 above)
Comment (saghm) [reply src/cmap/background.rs]: Unfortunately I think we're already set to do that, and it's failing to; see the default of the option [here](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#control_brace_style)

**Pull request merged**